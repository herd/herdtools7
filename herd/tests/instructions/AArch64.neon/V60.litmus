AArch64 V60
(* Test FMOV (to general) *)
{}
P0            ;
MOVI V0.4S,#1 ;
FMOV W0,S0    ;
forall 0:X0=1
