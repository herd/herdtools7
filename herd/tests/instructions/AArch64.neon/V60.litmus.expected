Test V60 Required
States 1
0:X0=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:X0=1)
Observation V60 Always 1 0
Hash=ca3dae5244d456916a35e023a99e71b9

